static/app/api.tsx

```
@@ -471,104 +471,112 @@ export class Client {
+          } else {
+            // There's no reason we should be here with a 200 response, but we get
+            // tons of events from this codepath with a 200 status nonetheless.
+            // Until we know why, let's do what is essentially some very fancy print debugging.
+            if (status === 200 && responseText) {
+              const parameterizedPath = sanitizePath(path);
+              const message = '200 treated as error';
+
+              const scope = new Sentry.Scope();
+              scope.setTags({endpoint: `${method} ${parameterizedPath}`, errorReason});
+              scope.setExtras({
+                twoHundredErrorReason,
+                responseJSON,
+                responseText,
+                responseContentType,
+                errorReason,
+              });
+              // Make sure all of these errors group, so we don't produce a bunch of noise
+              scope.setFingerprint([message]);
+
+              Sentry.captureException(
+                new Error(`${message}: ${method} ${parameterizedPath}`),
+                scope
+              );
+            }
+
+            const shouldSkipErrorHandler =
+              globalErrorHandlers.map(handler => handler(responseMeta)).filter(Boolean)
+                .length > 0;
+
+            if (!shouldSkipErrorHandler) {
+              errorHandler(responseMeta, statusText, errorReason);
+            }
           }
-        }
 
-        completeHandler(responseMeta, statusText);
-      })
-      .catch(() => {
-        // Ignore all failed requests
+          completeHandler(responseMeta, statusText);
+        },
+        () => {
+          // Ignore failed fetch calls or errors in the fetch request itself (e.g. cancelled requests)
+          // Not related to errors in responses
+        }
+      )
+      .catch(error => {
+        // eslint-disable-next-line no-console
+        console.error(error);
+        Sentry.captureException(error);
```

Due to https://github.com/getsentry/sentry/pull/34749 - we likely have been swallowing errors in response handlers 😰